### ISSUE TITLE
Clarity support for flutter 

### ISSUE THREAD

**Issue by alihassan143** [microsoft/clarity-apps#33]

**Comment by maryamshal** [comment]
We're thrilled to announce that the official **Clarity Flutter SDK** is now available for private preview, and it supports Session Recordings!
Be among the first to try it out by registering for the private preview through this [form](https://forms.microsoft.com/r/u1dRb8PsFa).

**Comment by Daeon97** [comment]
Hello @maryamshal we filled the form and have not gotten any response from the team yet. Is there a specific ETA between when we indicate interest in trying out the private preview and when we are able to start working with it

**Comment by dennymoradiya** [comment]
@maryamshal 

The recording is showing a blank screen, is it resolved or not?

I am using  `implementation 'com.microsoft.clarity:clarity:3.1.2'`

**Comment by dennymoradiya** [comment]
@maryamshal any update?

**Comment by Daeon97** [comment]
@dennymoradiya yes I have had access to clarity preview SDK for a while now thanks to @maryamshal assistance

**Comment by alihassan143** [comment]
@maryamshal any update when the plugin will be public beta or stable available?

**Comment by dennymoradiya** [comment]
@maryamshal any update?

**Comment by maryamshal** [comment]
Clarity Flutter SDK is still in private preview, stay tuned for the general release!
